eip155/caip25.md

```diff
@@ -34,7 +34,11 @@ see the [eip155/caip211.md](./caip211.md) profile for further guidance on using
 No namespace-wide or network-specific session properties have yet been proposed for standardization.
 When crafting such properties for contextual/in-network usage, it is recommended to align one's semantics and syntax (including case-sensitive style guides for property names!) with the [EIP-6963] wallet provider interface (which extends the [EIP-1193] interface) for common properties across architectures, making sure to avoid any collisions.
 
-Similarly, wherever possible, session properties should align closely with information passed as JSON objects by `wallet_` RPC methods, like capabilities or permissions. For example, the capability objects keyed to hexidecimal [EIP-155] `chainId`s defined as the expected return content of the `wallet_getCapabilities` method in [EIP-5792] should also be valid keyed the same way in `sessionProperties.capabilities`.
+Similarly, wherever possible, session properties should align closely with information passed as JSON objects by `wallet_` RPC methods, like capabilities or permissions. 
+For example, the capability objects specified in [EIP-5792] get requested per-address via the `wallet_getCapabilities` RPC method, and the returned objects are partitioned by the hexadecimal chainId when chain-specific, and inside of an object keyed `0x00` (following the chainId 0 convention; see the [`eip155`/caip2 profile](caip2)) when universal to the `eip155` namespace.
+Should an application request these capabilities and a wallet choose to pre-declare at time of connection, both parties can put such objects in the appropriately-keyed `scopedProperties` partition for chain-specific capabilities and in the `sessionProperties` (unpartitioned) for `eip155`-wide capabilities;
```

Comment:
> and in the `sessionProperties` (unpartitioned) for `eip155`-wide capabilities;

Hmmm I'm realizing that my gut feeling is that `sessionProperties` should be reserved for properties that live fully outside of any given namespace and that `scopedProperties` should allow you to map things to a namespace **or** a full CAIP-2 chainId scope.

Reply:
yeah, i feel you, but full CAIP-2 scopes are out, remember? hehe

Reply:
> full CAIP-2 scopes are out, remember
in what context? They're clearly still the keys in `scopedProperties`?

Reply:
doh i meant full-namespace scopes